client/my-sites/media-library/list.jsx

```diff
@@ -198,9 +201,11 @@ module.exports = React.createClass( {
 		var onFetchNextPage;
 
 		if ( this.props.filterRequiresUpgrade ) {
-			return (
-				<ListPlanPromo site={ this.props.site } filter={ this.props.filter } />
-			);
+			if( 'expanded' === abtest( 'expandedNudge' ) ) {
```

Comment:
I am aware of that, I thought I've read somewhere in our coding guidelines that they are optional. Removed them from this PR.